draft/draft-ietf-babel-yang-model.xml

```diff
@@ -173,22 +173,44 @@ INSERT_TEXT_FROM_FILE(../src/yang/ietf-babel@YYYY-MM-DD-abridged-tree.txt)
         next-hop address.</t>
 
         <t>Finally, for security two subtree are defined to contain MAC keys
-        and DTLS certificates. The mac-key-sets subtree contains keys used
-        with the MAC security mechanism. The boolean flag
-        babel-mac-default-apply indicates whether the set of MAC keys is
-        automatically applied to new interfaces. The dtls subtree contains
-        certificates used with DTLS security mechanism. Similar to the MAC
-        mechanism, the boolean flag babel-dtls-default-apply indicates whether
-        the set of DTLS certificates is automatically applied to new
+        and DTLS certificates. The mac subtree contains keys used with the MAC
+        security mechanism. The boolean flag default-apply indicates whether
+        the set of MAC keys is automatically applied to new interfaces. The
+        dtls subtree contains certificates used with DTLS security mechanism.
+        Similar to the MAC mechanism, the boolean flag default-apply indicates
+        whether the set of DTLS certificates is automatically applied to new
         interfaces.</t>
       </section>
 
       <section title="YANG Module">
-        <t>This module augments <xref target="RFC8343">A YANG Data Model for
-        Interface Management </xref>, <xref target="RFC8349">YANG Routing
-        Management </xref>, imports definitions from <xref
-        target="RFC6991">Common YANG Data Types</xref>, and references <xref
-        target="RFC2104">HMAC: Keyed-Hashing for Message
+        <t>This YANG module augments the <xref target="RFC8349">YANG Routing
+        Management</xref> module to provide a common framework for all routing
+        subsystems. By augmenting the module it provides a common building
+        block for routes, and Routing Information Bases (RIBs). It also has a
+        reference to an interface defined by <xref target="RFC8343">A YANG
+        Data Model for Interface Management</xref>.</t>
+
+        <t>A router running Babel routing protocol can determine the
+        parameters it needs to use for an interface based on the interface
+        name. For example, it can detect that eth0 is a wired interface, and
+        that wlan0 is a wireless interface. This is not true for a tunnel
+        interface, where the link parameters need to configured explicitly.
```

Comment:
s/need to configured/need to be configured/

Reply:
Ok